src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/request.cache.max-age=0.and.304/connect/server.rpt

```diff
@@ -29,6 +29,7 @@ read nukleus:begin.ext ${http:header(":scheme", "http")}
                        ${http:header(":method", "GET")}
                        ${http:header(":path", "/path")}
                        ${http:header(":authority", "localhost:8080")}
+                       ${http:header("cache-control", "max-age=0")}
```

Comment:
I don't believe this is needed as the point of the tests is to test max-age=0 on the second request...?

Reply:
yeap doesn't make any difference.

Reply:
I just checked it `max-age=0` is sent from the first request as well.

Reply:
I see its on the others.  leaving it